internal/exec/stages/disks/disks.go

```diff
@@ -265,3 +275,35 @@ func (s stage) createFilesystem(fs types.FilesystemMount) error {
 
 	return nil
 }
+
+func (s stage) readFilesystemType(fs types.FilesystemMount) (string, error) {
+	var fsType string
+	err := s.Logger.LogOp(
+		func() error {
+			blkid := "blkid"
+			args := []string{"-ovalue", "-sTYPE", string(fs.Device)}
+			stdout := &bytes.Buffer{}
+			cmd := exec.Command(blkid, args...)
```

Comment:
All give you that this is maybe a little shorter than using libblkid but personally I would favor that over calling blkid